react/features/settings/components/web/MoreTab.js

```diff
@@ -50,6 +50,11 @@ export type Props = {
      */
     languages: Array<string>,
 
+    /**
+     * The types of notifications that can be configured and their specific visibility.
+     */
+    notifications: Object,
```

Comment:
same, rename to `enabledNotifications` -it will be renamed when you rename in the `getMoreTabProps` props function